lib/tasks/setup_s3.rake

```diff
@@ -0,0 +1,132 @@
+namespace :heroku do
```

Comment:
Please use `deploy` as namespace, also move this script to `lib/tasks/deploy` folder. Rename script to `create-s3-bucket` rake tasks.

Create sh script `bin/setup-s3`, this should take AWS parameters and pass them to rake task. After bucket is created, use `heroku` to set variables on heroku.